### ISSUE TITLE
Cannot load wallet (1): Wallet format requires update. Cannot find keystore for name x1/

### ISSUE THREAD

**Issue by calhroub** [spesmilo/electrum#4355]
Hi. I am having the same issue but on Mac OS X El Capitan. Currently Even when updating to the latest Electrum 3.1.3 I keep getting this error:
> Cannot load wallet (1):
> Wallet format requires update.
> Cannot find keystore for name x1/

https://i.imgur.com/DsRyCeg.png

It was working fine until last night when I was trying to import an old Dark Wallet seed and private keys of a multi-sig address created from 3 stealth addresses... Any idea how to fix this, would be appreciated. Thanks.

I figured I should create a new issue because #2639 seems to be closed and was for Linux Electrum wallet...

**Comment by SomberNight** [comment]
Hi.

> It was working fine until last night when I was trying to import an old Dark Wallet seed and private keys of a multi-sig address created from 3 stealth addresses...

I don't really understand.. do you have a way to reproduce the error? As in, to create a wallet file that produces it.

Otherwise, to be able to select a different wallet, you could change the name of the wallet file it is trying to open.
I've pushed a fix to master (https://github.com/spesmilo/electrum/commit/f0657a35071d74e7ee0e0897601dc670709cb907) so that the wizard would appear in this case.

**Comment by calhroub** [comment]
Hi. Regarding more info about the Dark Wallet issue that I'm having: https://www.reddit.com/r/BitcoinWallet/comments/6ygi9w/issues_with_dark_wallet_do_they_have_product/

> do you have a way to reproduce the error? As in, to create a wallet file that produces it.
I'm not sure. Could you walk me through that? Or point me to some instructions? Usually Mac gives me an error log if an app fails, but in this case it just gave me that error in the screenshot...

> Otherwise, to be able to select a different wallet, you could change the name of the wallet file it is trying to open.
How do I do that? Sorry, I'm not very technically savvy with bitcoin stuff

> I've pushed a fix to master (f0657a3) so that the wizard would appear in this case.
What can I do with this?

Thank you very much for your reply.

**Comment by calhroub** [comment]
Thanks. I found the wallet that was created last, changed it's name and was able to open my other electrum wallets.

> I would like to figure out how you created the faulty wallet in the first place. I would need more info/description from you to do that.

Sorry, I had linked the wrong reddit link in the previous reply. I modified it to the correct one: https://www.reddit.com/r/Bitcoin/comments/8i1ev8/help_please_i_have_btc_stuck_in_dark_wallet_080/

Basically, I have the seed of this old Dark Wallet 3-of-3 multi-sig address created from 3 stealth addresses. I have tried many paths. One of them that caused the electrum wallet issue I was having was I tried to import the multi-sig address. I'm not quite sure what I did, because I never tried importing a multisig wallet. Now that I am recreating the problem, I believe selected the restore -> Multi-sig wallet -> 1 from 3 (instead of 3-from-3) -> I already have seed -> put my seed in (below seed box it changed to Seed type: old) -> options -> BIP39 seed -> Next -> Derivation path: m/44'/0'/0' -> it gave me  Master public key -> asked me for cosigner seed or private key options. At this point I'm not sure what I entered because it won't accept my private keys...

I've been battling this Dark Wallet issue for a while. Lost many battles. Gave up a few times, then tried again, etc... Sorry if that wasn't very helpful. I'm currently very exhausted and frustrated with this ded Dark Wallet... Thank you very much for helping me. Feel free to ask anything, and I'll do my best to provide feedback.
